### PR TITLE
Model updates, new features and feature eng

### PR DESCRIPTION
**Pull request by DivLoic**

### CODE REVIEW

File: edml-trainer/trainer/util.py [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-TABLE_ID = "tlc_yellow_trips_2018"
+DATASET_ID = "edml_nyc_yellow_taxi_us"
+TABLE_ID = "gis_feat_eng"
 
-CSV_COLUMNS = ["uuid", "dayofweek", "hourofday", "pickup_zone_name", "dropoff_zone_name", "passenger_count", "trip_duration"]
+CSV_COLUMNS = ["uuid", "dayofweek", "hourofday", "weekofyear", "pickup_zone_name", "dropoff_zone_name", "passenger_count", "distance", "trip_duration"]
 LABEL_COLUMN = "trip_duration"
 KEY_COLUMN = "uuid"
 
 # Set default values for each CSV column
```
Comment (DivLoic) [inline edml-trainer/trainer/util.py:14]: @giulbia I'm not sure you meant _"csv"_.

File: edml-trainer/trainer/util.py [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-DEFAULTS = [['no_key'], [1], [0], [""], [""], [1], []]
+DEFAULTS = [['no_key'], [1], [0], [54], [""], [""], [1], [0], []]
```
Comment (DivLoic) [inline edml-trainer/trainer/util.py:15]: @giulbia this is default what?

Comment (DivLoic) [inline edml-trainer/trainer/util.py:15]: @giulbia magic numbers, please map them to their column.

File: edml-trainer/requirements.txt [hunk 3]
```diff
@@ -1,8 +1,2 @@
-pandas
-sklearn
-scikit-learn
-scipy
 tensorflow==1.15.2
-tensorflow-io==0.8.1
-
-pytest==5.4.1
\ No newline at end of file
+tensorflow-io==0.9.1
```
Comment (DivLoic) [inline edml-trainer/requirements.txt]: @giulbia to be reverted :/

File: edml-trainer/requirements.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
 tensorflow==1.15.2
```
Comment (DivLoic) [inline edml-trainer/requirements.txt:1]: @giulbia please add `pytest` back and pint it to the latest version

Comment (giulbia) [reply edml-trainer/requirements.txt:1]: 5.4.1

File: edml-trainer/trainer/util.py (hunk 1 above)
Comment (giulbia) [reply edml-trainer/trainer/util.py:14]: definitely not anymore

File: edml-trainer/trainer/util.py (hunk 2 above)
Comment (giulbia) [reply edml-trainer/trainer/util.py:15]: it's the default values for each column in case there are missing values in input data

Comment (giulbia) [reply edml-trainer/trainer/util.py:15]: not magic numbers. what do you mean? do you want a dictionary instead?

**Pull request merged**